caliban/util/argparse.py

```diff
@@ -39,6 +40,25 @@ def expand_args(items: Dict[str, str]) -> List[str]:
   return list(it.chain.from_iterable(pairs))
 
 
+def argparse_schema(schema):
```

Comment:
Typing annotations would be helpful for me here, as I'm new to schema. 
Would this be something along the lines of: (?)

SchemaType = Union[s.And, s.Or, s.Regex, s.Use, s.Schema, s.Const] 
def argparse_schema(schema: SchemaType) -> Callable[Any, [Any]]: